### DISCUSSION TITLE
How to contribute localizing the project (chrome://ungoogled-first-run/, etc.)

### DISCUSSION THREAD

**Discussion by bywkgexmkiltxh** [Eloston/ungoogled-chromium#2901] [General]
Adding native languages for non-English speakers might help them understanding the gist of the text and catalyze installing the extensions

**Comment by rany2** [comment]
There are languages other than English?! /s

But right now, there is no easy way to localize it. It would require reworking https://github.com/ungoogled-software/ungoogled-chromium/blob/edd233e70275def84d6648604fe5169d39cbd43c/patches/extra/ungoogled-chromium/first-run-page.patch

**Reply by bywkgexmkiltxh** [reply]
If it's hard, it's not worth it, but I participate in the translations for some software using Crowdin, [localise.biz](https://localise.biz/), and they work somehow, so I thought it was up to implementing to these localization services

**Reply by rany2** [reply]
Well, it might be possible in the future but right now it's essentially just a static HTML page.
